vhost_user_net/src/lib.rs

```diff
@@ -184,7 +188,13 @@ impl VhostUserBackendMut<VringRwLock<GuestMemoryAtomic<GuestMemoryMmap>>, Atomic
             | 1 << VIRTIO_NET_F_MAC
             | 1 << VIRTIO_F_NOTIFY_ON_EMPTY
             | 1 << VIRTIO_F_VERSION_1
-            | VhostUserVirtioFeatures::PROTOCOL_FEATURES.bits()
+            | VhostUserVirtioFeatures::PROTOCOL_FEATURES.bits();
+
+        if self.mtu.is_some() {
```

Comment:
This feature should always be advertised - we can always get an MTU from the tap.

Reply:
Yep!